src/aks-preview/setup.py

```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "0.5.122"
+VERSION = "0.5.123"
```

Comment:
Please also update history.rst to describe your changes.

Reply:
Added